src/sqlfluff/dialects/dialect_athena.py

```diff
@@ -0,0 +1,541 @@
+"""The AWS Athena dialect.
+
+https://docs.aws.amazon.com/athena/latest/ug/what-is.html
+"""
+
+from sqlfluff.core.dialects import load_raw_dialect
+from sqlfluff.core.parser import (
+    AnyNumberOf,
+    BaseSegment,
+    Bracketed,
+    Delimited,
+    NamedParser,
+    Nothing,
+    OneOf,
+    OptionallyBracketed,
+    Ref,
+    RegexParser,
+    Sequence,
+    StringLexer,
+    StringParser,
+    SymbolSegment,
+)
+from sqlfluff.core.parser.grammar.anyof import AnySetOf
+from sqlfluff.core.parser.segments.raw import CodeSegment, KeywordSegment, RawSegment
+from sqlfluff.dialects import dialect_ansi as ansi
+from sqlfluff.dialects.dialect_athena_keywords import (
+    athena_reserved_keywords,
+    athena_unreserved_keywords,
+)
+
+ansi_dialect = load_raw_dialect("ansi")
```

Comment:
IIUC, Presto has a lot of overlap with Hive. Should we consider basing Presto dialect on Hive rather than ANSI? (Not a recommendation, just a question.)

Reply:
I thought about that, but I think I was biased by tunetheweb last comment on issue #2168 and chose to start with ansi and build from there. Maybe going forward I can experiment using hive as base.

Reply:
Nice. Good to know that this can be done.